packages/api/src/controllers/stream.ts

```diff
@@ -888,11 +893,13 @@ app.put(
 
     // trigger the webhooks, reference https://github.com/livepeer/livepeerjs/issues/791#issuecomment-658424388
     // this could be used instead of /webhook/:id/trigger (althoughs /trigger requires admin access )
-    await req.queue.publish("events.streams", {
+    const event = req.body.active === true ? "stream.started" : "stream.idle";
+    await req.queue.publish(`events.${event}`, {
+      type: "webhook_event",
       id: uuid(),
-      createdAt: Date.now(),
-      channel: "webhooks",
-      event: req.body.active === true ? "stream.started" : "stream.idle",
+      timestamp: Date.now(),
+      manifestId: stream.playbackId,
```

Comment:
nowadays, in our codebase, `manifestId` can mean many different things. 
If we're passing `playbackId` - maybe it's better to call it `playbackId`? That (still 😄 ) has only one meaning 😄

Reply:
Ideally, here we would use some kind of "global identifier" which is known by all Livepeer services and uniquely identifies each stream in our infrastructure. So that a service processing all the messages on RabbitMQ would be able to join all events from a given stream using only the ID value in that respective field.

Since `go-livepeer` apparently has no reference to `playbackId`s ([only one I found was a comment for a field that is actually `ManifestID`](https://github.com/livepeer/go-livepeer/blob/72047f1d2d95b89a308da1064f3f3be08c89d1ec/core/streamdata.go#L165)), I __think__ it makes more sense to stick with `manifestId` for the API-agnostic field here, unless you have some other idea. Some alternatives that came to my mind but are more like questions:
 - Could we assume on `go-livepeer` that every `manifestId = playbackId` and set a field called `playbackId` with the value known there as `manifestId` instead? So in the events we have `playbackId`, and on `go-livepeer` we set `event.playbackId = mid`, instead of the opposite here
 - Do we have a way to determine the "true" `manifestId` here? That since it can be set to different things, not only the `playbackId`, is that right? Or is it truly non-deterministic for the same stream, where different services use different values but call all of them `manifestId`?

I guess the more open questiion is: What's the best way to create this "global identifier" to be used by a service joining all events about a given stream?

Reply:
Resolved by using the `streamId` as the global stream identifier.

livepeer-data:
 - https://github.com/livepeer/livepeer-data/pull/7/commits/029683482b09a135f5b13a620213b43ec6dfb55c
go-livepeer:
 - https://github.com/livepeer/go-livepeer/pull/2002/commits/b814dfb08208f1857b7bb6e01ac448246b006140
livepeer-com:
 - https://github.com/livepeer/livepeer-com/pull/569/commits/74f1a4cb2a055be10fbf29b1db87bdb74ab7f02a